examples/solo5-irmin/dispatch.ml

```diff
@@ -6,56 +6,78 @@ module type HTTP = Cohttp_lwt.S.Server
 
 (* Logging *)
 let http_src = Logs.Src.create "http" ~doc:"HTTP server"
+
 module Http_log = (val Logs.src_log http_src : Logs.LOG)
 
 module BlockCon = struct
-  include Block (* from mirage-block-solo5 *)
-  let discard _ _ _ = Lwt.return @@ Ok ()
-end
+  include Block
 
-module DB = Wodan_irmin.DB_BUILDER
-  (BlockCon)(Wodan.StandardSuperblockParams)
-module Wodan_Git_KV = Wodan_irmin.KV_git(DB)
+  (* from mirage-block-solo5 *)
```

Comment:
Moving the comment two lines down changes its meaning.
@gpetiot, is it expected that https://github.com/ocaml-ppx/ocamlformat/commit/2de7c5f would move comments?
Do you recommend a different snapshot?

Reply:
Good catch, I fixed it here.

Reply:
Reported in https://github.com/ocaml-ppx/ocamlformat/issues/660